packages/runner/injection/multi-domain.js

```diff
@@ -40,4 +16,19 @@ const cyBridgeFrame = window.parent.frames[2]
 
 const Cypress = cyBridgeFrame.Cypress
 
+if (!Cypress) {
+  throw new Error('Something went terribly wrong and we cannot proceed. We expected to find the global \
```

Comment:
I think throwing this error is ok for now, but we definitely need to revisit this (and we have a story to do so) but if cypress is not found the page load event just hangs and times out, the thrown error doesn't make it back to main cypress. I ran into this too in my story.